control-plane/aegisops_control_plane/service.py

```
@@ -4911,10 +4911,90 @@ def run_live_assistant_workflow(
+            adapter=adapter,
+        )
+        return snapshot
+
+    def _persist_live_assistant_feedback_loop(
+        self,
+        *,
+        record_family: str,
+        record_id: str,
+        context_snapshot: AnalystAssistantContextSnapshot,
+        workflow_snapshot: LiveAssistantWorkflowSnapshot,
+        provider_result: AssistantProviderResult | AssistantProviderFailure | None,
```

Ruff (0.15.10)

[error] Undefined name `AssistantProviderResult`

(F821)